Extensions/blacklist.js

```diff
@@ -160,7 +167,7 @@ XKit.extensions.blacklist = new Object({
 						" position: absolute; top: 0; left: 0; width: 100%; " +
 						" color: rgba(255,255,255,.43); height: 27px !important; padding: 0px; !important; " +
 						" line-height: 27px !important; padding-left: 15px; !important; } " +
-					" .xblacklist_blacklisted_post .post_content { " +
+					` .xblacklist_blacklisted_post ${postContentSel} { ` +
```

Comment:
so when it's used here, the rule becomes

```css
.xblacklist_blacklisted_post ._2OqKn, ._2-mU3, ._1yQrs, ._1Wz4U {
	background: transparent;
	color: rgba(255,255,255,.43);
}
```

which targets a lot more than intended and breaks the page.